src/js/consts/mimeType.js

```diff
@@ -0,0 +1,6 @@
+const MimeType = {
+  XLS: 'application/vnd.ms-excel',
+  PDF: 'application/pdf',
```

Comment:
would it be worth merging this with fileFormat given that (based on your code) it's requirement that they have the same keys? Could just have a single enum with multiple fields. I have no idea if this is even possible with javascript :sweat_smile: